docs/cellular_automata_module.md

```
@@ -0,0 +1,218 @@
+   `namespace Aleph::CA`. No se introducen nuevas dependencias externas
+   obligatorias.
+
+3. **Reuso de Aleph-w** — siempre que sea posible, se construye sobre
+   piezas existentes (`tpl_dynArray.H`, `bitArray.H`, `thread_pool.H`,
+   `ah-parallel.H`, `ah-errors.H`, …).
+
+---
+
+## 2. Diagrama de capas
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)